test-output/Default Suite/Procyon-QAC-G1-13.10.22.html

```diff
@@ -0,0 +1,95 @@
+<html>
```

Comment:
Please remove all of the test-output directory and add this test-output directory to .gitignore file in order to not be checked in GitHub. Thanks!

Reply:
Is this the syntax I need when adding the directory to .gitignore?
test-output/